packages/actor-http-fetch/test/ActorHttpFetch-test.ts

```diff
@@ -443,5 +443,67 @@ This error can be disabled by modifying the 'httpBodyTimeout' and/or 'httpTimeou
       );
       jest.useRealTimers();
     });
+
+    it('should retry with a delay', async() => {
+      let numberOfRetries = 0;
+      const customFetch = jest.fn(async(_, init) => {
+        if (numberOfRetries < 2) {
+          numberOfRetries++;
+          throw new Error('Retry count not reached.');
+        }
+        return {};
+      });
+
+      // Time delay time.
+      const start = performance.now();
+      await actor.run({
+        input: <Request> { url: 'ignored by custom fetch' },
+        context: new ActionContext({
+          [KeysHttp.fetch.name]: customFetch,
+          [KeysHttp.httpRetryCount.name]: 2,
+          [KeysHttp.httpRetryDelay.name]: 100,
+        }),
+      });
+
+      const time = performance.now() - start;
+      expect(time).toBeGreaterThan(200);
+      expect(time).toBeLessThan(250);
```

Comment:
What guarantees it being lest than `250`?

Reply:
The rest of the test does not take more than 10 ms at max (basically just calls the mockedFetch). So I thought, it might be reasonable to expect the whole test to be within that bounds and larger time spans indicate that something is going wrong.

Reply:
I think leave it out just because the tests suites are run concurrently and so is a little unpredictable and has the potential to become quite flakey as a test and cause workflow failures.